tests/integration/vectors_community/json/statedump.json

```diff
@@ -842,7 +842,7 @@
                     }
                 ],
                 "storage": {
-                    "0x81e47a19e6b29b0a65b9591762ce5143ed30d0261e5d24a3201752506b20f15c": "0xe9ac0c50"
+                    "0x81e47a19e6b29b0a65b9591762ce5143ed30d0261e5d24a3201752506b20f1": "0xe9ac0c50"
```

Comment:
This state dump was taken before the change to 31 byte state keys, in case you're wondering. :)